### ISSUE TITLE
Conda activate and deactivate no longer appearing

### ISSUE THREAD

**Issue by diego898** [esc/conda-zsh-completion#14]
I just installed this, but I still can't get activate and deactivate to show up:

![image](https://user-images.githubusercontent.com/742407/54091162-93ee0000-434a-11e9-9f77-19895b3b9579.png)

**Comment by esc** [comment]
Can you list how you installed the completion, which version it is and the output of `print -l $fpath`.

**Comment by diego898** [comment]
Im using the latest version, installed using the instructions on the top of the file:

**Installation and use:**
![image](https://user-images.githubusercontent.com/742407/54092152-b71eac80-4356-11e9-8cc4-6960392e99b6.png)

**My current fpath:**

```
➜  praxis ~/bin print -l $fpath
/home/mesada/.oh-my-zsh/plugins/python
/home/mesada/.oh-my-zsh/custom/plugins/zsh-syntax-highlighting
/home/mesada/.oh-my-zsh/plugins/fzf
/home/mesada/.oh-my-zsh/plugins/colorize
/home/mesada/.oh-my-zsh/plugins/sudo
/home/mesada/.oh-my-zsh/plugins/history
/home/mesada/.oh-my-zsh/plugins/git
/home/mesada/.oh-my-zsh/functions
/home/mesada/.oh-my-zsh/completions
/usr/local/share/zsh/site-functions
/usr/local/share/zsh/5.7.1/functions
/home/mesada/bin/conda-zsh-completion
```

**Comment by esc** [comment]
Could it be, that an older version of the completion is installed in home/mesada/bin/conda-zsh-completion?

**Comment by diego898** [comment]
@esc I don't think so - the first line in my screenshot is me pulling down a fresh clone into that directory

**Comment by esc** [comment]
@diego898 I am out of ideas, sorry.

**Comment by johnstill** [comment]
@diego898, I installed this autocomplete to `/usr/local/share/zsh/site-functions` when I installed zsh 5.7 on praxis.   This dir is ahead of `/home/mesada/bin/conda-zsh-completion` in your `fpath` and hence was overriding your installation.  If you have custom zsh functions they need to be at the head of your `fpath` or they may be shadowed.  I've reinstalled this completion func with the updated version (looks like `activate` support was just added a few days ago); I'll follow up with you offline.
